ui/app/components/app/account-menu/account-menu.component.js

```diff
@@ -37,35 +39,74 @@ export default class AccountMenu extends PureComponent {
     toggleAccountMenu: PropTypes.func,
   }
 
+  accountsRef;
+
   state = {
     atAccountListBottom: false,
+    searchQuery: '',
   }
 
+  addressFuse = new Fuse([], {
+    shouldSort: false,
+    threshold: 0.45,
+    location: 0,
+    distance: 100,
+    maxPatternLength: 32,
+    minMatchCharLength: 1,
+    keys: [
+      { name: 'name', weight: 0.5 },
+      { name: 'address', weight: 0.5 },
+    ],
+  })
+
   componentDidUpdate (prevProps) {
-    const { prevIsAccountMenuOpen } = prevProps
+    const { isAccountMenuOpen: prevIsAccountMenuOpen } = prevProps
```

Comment:
it's probably a typo as there is no `prevIsAccountMenuOpen` prop passed anywhere

Reply:
Nice catch! It does look like a typo.